src/extensions/arith.rs

```diff
@@ -405,21 +529,21 @@ impl Expr<CovExtArgs> {
                 let (x, sat_x) = x.satisfy(env, s)?;
                 let (y, sat_y) = y.satisfy(env, s)?;
                 let res = x.checked_add(y).ok_or(EvalError::AddOverflow(x, y))?;
-                let sat = Satisfaction::combine(sat_x, sat_y);
+                let sat = Satisfaction::combine(sat_y, sat_x);
```

Comment:
In 5e533b71c8904f8ad2e0f6ec7ec50f33e3ecd13a:

Lol, I guess none of these mattered before because none of our arithmetic fragements took witnesses?